src/db.c

```diff
@@ -219,32 +220,33 @@ void dbAdd(serverDb *db, robj *key, robj *val) {
     dbAddInternal(db, key, val, 0);
 }
 
-/* Returns key's hash slot when cluster mode is enabled, or 0 when disabled.
- * The only difference between this function and getKeySlot, is that it's not using cached key slot from the
- * current_client and always calculates CRC hash. This is useful when slot needs to be calculated for a key that user
- * didn't request for, such as in case of eviction. */
-int calculateKeySlot(sds key) {
-    return server.cluster_enabled ? keyHashSlot(key, (int)sdslen(key)) : 0;
+/* Returns which db index should be used with kvstore for a given key. */
+static int getKVStoreIndexForKey(sds key) {
+    return server.cluster_enabled ? getKeySlot(key) : 0;
```

Comment:
So this gets inlined now, for non-cluster mode?

What about cluster mode, would it make sense to inline getKeySlot or parts of it too?

Again I see "db index" in the comment, not a good name, since it's not the DB index as in SELECT.

Reply:
With clang on x86 and arm, getKVStoreIndexForKey is getting inlined in all of the call sites. 

I did some testing with forcing the inline of getKeySlot and wasn't to see any difference.

Reply:
:ok: